staging/src/k8s.io/kms/apis/v2beta1/api.proto

```diff
@@ -17,8 +17,8 @@ limitations under the License.
 // To regenerate api.pb.go run `hack/update-codegen.sh protobindings`
 syntax = "proto3";
 
-package v2alpha1;
-option go_package = "k8s.io/kms/apis/v2alpha1";
+package v2beta1;
+option go_package = "k8s.io/kms/apis/v2beta1";
```

Comment:
Maybe we should keep the `status.version` field and just remove the sub version stuff from here?

```suggestion
option go_package = "k8s.io/kms/apis/v2";
```

Reply:
I think I agree with this and basically said the same thing at https://github.com/kubernetes/kubernetes/pull/115123#discussion_r1136285848